sitemap.md

```diff
@@ -1,8 +1,10 @@
  * [Home](index.md)
  * [User's Guide](users-guide.md)
+   * [Dagger & Android](android.md)
    * [Multibindings](multibindings.md)
    * [Subcomponents](subcomponents.md)
    * [Migrating from Dagger 1](dagger-1-migration.md)
    * [Producers](producers.md)
+   * [Testing](testing.md)
```

Comment:
Ditto.  We can follow up.  I'm trying to figure out a way of getting a jekyll template that can interpret these sitemap/navbar md files, but for now, these are strictly irrelevant to the open-source docs.